docs/user-event/options.mdx

```diff
@@ -0,0 +1,118 @@
+---
+id: options
+title: Options
```

Comment:
What isn't clear to me are what these options are.
Are these the options for `setup`? 
If that's the case, what about adding them to the `setup` page or to make it explicit that these are render options in an intro text?

Reply:
I think one of the biggest mistakes of the previous docs was to mention the same options at different locations and splatter them across the docs.

If you know that you should use `setup` and how you should use `pointer` and `keyboard`, I think the other APIs are pretty much self-explanatory.
But one can use the library for months without ever needing any option and the options might also be extended in the future. So I think a prominent place in the docs is a better choice than nesting them under `setup`. (Also IMHO the docs design works much better with short pages.)

Intro text to make this more clear :+1:

Reply:
Added an intro text.